asset_dashboard/static/js/PortfolioPlanner.js

```diff
@@ -332,25 +405,40 @@ class PortfolioPlanner extends React.Component {
     }))
   }
 
-  createRegionName(regions) {
-    // returns a CSV string of names for the different regions
-    const names = regions.map(({ name }) => {
-      return name
-    }).join(',')
-
-    return names
-  }
-
   getDate() {
     const date = new Date(new Date().toString().split('GMT')[0]+' UTC').toISOString().split('T')[0]
     return date
   }
+  
+  changeSection(e) {
+    const newSection = e.target.value
+    this.setState(prevState => {
+      return {
+        ...prevState,
+        selectedSection: newSection
+      }
+    })
+  }
 
   render() {
-    // This filters on every re-render, so that this.state.remainingProjects can be the source of truth.
-    // Could also chain filtering here for other things (like "department" from the wireframe).
     const filteredRows = this.state.remainingProjects && this.state.remainingProjects.filter(project => {
       return project.description.toLowerCase().includes(this.state.filterText.toLowerCase())
+    }).filter(project => {
+      if (this.state.selectedSection !== '') {
+        return project.section.toLowerCase().includes(this.state.selectedSection.toLowerCase())
+      } else {
+        return project
+      }
+    })
+    
+    const filteredPortfolio = this.state.portfolio.projects && this.state.portfolio.projects.filter(project => {
+      return project
+    }).filter(project => {
+      if (this.state.selectedSection !== '') {
+        return project.section.toLowerCase().includes(this.state.selectedSection.toLowerCase())
```

Comment:
you've used this construct a few times.

what about creating a little helper function

```
function caseInsensitiveIncludes(source, target) {
     return source.toLowerCase().includes(target.toLowerCase())
```